src/parser/ast.rs

```diff
@@ -15,19 +15,21 @@ pub enum Statement {
     ConstantDefinition(String, Expression),
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq, Clone)]
 pub struct SelectedExpressions {
     pub selector: Option<Expression>,
     pub expressions: Vec<Expression>,
 }
 
-#[derive(Debug, PartialEq, Eq)]
+// TODO use bignum or something
```

Comment:
I've been using `U256` from `ethers_core`, but not sure if it's overkill.

Reply:
I think it should really be a bignum.